defaults/config.js

```diff
@@ -86,6 +86,28 @@ module.exports = {
 	//
 	prefetchStorage: false,
 
+	//
+	// Allow uploading files to the server hosting TheLounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/storage` folder.
+	//
+	// Images will not expire or be removed by TheLounge, so be wary of memory and your hardware.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	upload: true,
+
+	//
+	// Maximum file size for uploaded files
+	//
+	// If upload is enabled, The Lounge will only allow uploading files with a size < the value below
+	// Specified value is in kilobytes. Default value is 1e7 bytes (10 MB).
+	//
+	// @type     int
+	// @default  1e7
+	//
+	uploadMaxFileSize: 1e7,
```

Comment:
10mb would be 10240.

1e7 is 10 gigabytes.

Reply:
its in bytes not KB

Reply:
Your comment says kilobytes.

The other max size we have in the config is also KB, so do the math in the code.